.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+    -
+        repo: 'https://github.com/ambv/black'
+        # 18.6b1
+        rev: ed50737290662f6ef4016a7ea44da78ee1eff1e2
+        hooks:
+            - id: black
+              args: ['--safe']
+              language_version: python3.6
+    -
+        repo: 'https://github.com/pre-commit/pre-commit-hooks'
+        rev: v2.0.0
+        hooks:
+            - id: flake8
+              language_version: python3.6
+              args: [
+                  # E501 let black handle all line length decisions
+                  # W503 black conflicts with "line break before operator" rule
+                  # E203 black conflicts with "whitespace before ':'" rule
+                  # E722 bare excepts need to be addressed
```

Comment:
These should eventually be addressed, though I think both cases in this repo _should_ actually raise instead of excepting, printing an error, then exiting.